Jenkinsfile

```diff
@@ -13,32 +13,19 @@ pipeline {
         DEBUG = 0
     }
     options {
+        retry(1)
```

Comment:
to future historians: added a retry not because our code is flaky but the internet in general has a habit of failing exactly when the end to end smoke tests require it to not